client/rpcserver/handlers.go

```diff
@@ -252,6 +252,15 @@ func handleRegister(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 	return createResponse(registerRoute, &resp, nil)
 }
 
+// format concatenates thing and tail. If thing is empty, returns an empty
+// string.
+var format = func(thing, tail string) string {
```

Comment:
`func format(...`?

Reply:
gah. srry. was inside a function then I moved it out.